domain/transfer_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/dogmatiq/example/messages"
 	"github.com/dogmatiq/example/messages/commands"
 	"github.com/dogmatiq/example/messages/events"
+	"github.com/dogmatiq/testkit"
```

Comment:
It's common in go tests to "dot import" the test dependencies; you could do that here with `testkit`.

Reply:
I did the dot import as suggested.